domains/aaronfort.json

```
@@ -0,0 +1,11 @@
+{
+  "description": "Documentation website for Aaron Fort",
+  "repo": "https://github.com/AaronFortG",
+  "owner": {
+    "username": "AaronFortG",
+    "email": "aaronfortg@gmail.com"
+  },
+  "record": {
+    "CNAME": "https://mycurriculum.salleurl.edu/aaron-fort/"
```

Ya need to remove 'https://' and '/aaron-fort/', that's not supported by CNAME records, and by any records either (Excepted URL records, but they redirect instead of really having a domain.)
```suggestion
    "CNAME": "mycurriculum.salleurl.edu"
```